test_integration/lib/test/test_names.dart

```diff
@@ -11,8 +11,10 @@ class TestName {
       'testHelperUnhandledExceptionTest';
 
   static const String restPublish = 'restPublish';
+  static const String restHistory = 'restHistory';
   static const String restPublishWithAuthCallback =
       'restPublishWithAuthCallback';
+  // TODO(tiholic) handle restHistoryWithAuthCallback
```

Comment:
Again, it looks like we should manage this deferred work with an issue.

Reply:
https://github.com/ably/ably-flutter/issues/86

Reply:
https://github.com/ably/ably-flutter/issues/87